options.html

```diff
@@ -31,13 +31,24 @@
                 <button id="save" data-message="optionsSave">Save</button>
             </div>
         </div>
-        <div class="row hidden" id="manualEntry">
-            <div class="col-xs-2" data-message="optionsShortCodeLabel">Short code:</div>
-            <div class="col-xs-8">
-                <input type="text" id="tenantCode" />
+        <div class="hidden" id="manualEntry">
+            <div class="row">
+                <div class="col-xs-2" data-message="optionsShortCodeLabel">Short code:</div>
+                <div class="col-xs-2">
+                    <input type="text" id="tenantCode" />
+                </div>
             </div>
+            <div class="row">
+                <div class="col-xs-2" data-message="optionsRegionLabel">Region:</div>
```

Comment:
Same thing here about this being an actual `<label>` rather than a div made to look like a label.